data/strings/types_strings.txt

```diff
@@ -1138,7 +1138,7 @@
     zh-Hant = 自行車充電站
 
   [type.amenity.charging_station.motorcar]
-    en = Motorcar Charging Station
+    en = Car Charging Station
```

Comment:
In the US,we call them cars.  In en_GB, I think motorcar is perhaps right.

Reply:
we may be silly in the UK, but we don't call them motorcars anymore!